src/ZoneServer/Pads/Handlers/Swordsman/Hoplite/ThrowingSpear_Hoplite33.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Threading.Tasks;
+using Melia.Shared.Game.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Melia.Zone.World.Actors.Monsters;
+using Melia.Zone.World.Actors.Pads;
+using Yggdrasil.Logging;
+
+namespace Melia.Zone.Pads.Handlers.Swordsman.Hoplite
+{
+	/// <summary>
+	/// Handler for the ThrouwingSpear_Hoplite33_Pad,
+	/// which displays an explosion effect in a given position
+	/// </summary>
+	[PadHandler("ThrouwingSpear_Hoplite33_Pad")]
+	public class ThrouwingSpear_Hoplite33_Pad : ICreatePadHandler, IUpdatePadHandler, IDestroyPadHandler
+	{
+		/// <summary>
+		/// Called when the pad is created.
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="args"></param>
+		public void Created(object sender, TriggerArgs args)
```

Comment:
I'd appreciate it if you could update your pad handling for the latest version of master, where handlers not use `PadTriggerArgs` and subscriptions should be done via the `Subscribe` method in most cases. Once this is done, you also don't need to check the types anymore, as you currently are below.

Reply:
Okay, hopefully this is right.